TaskManager.java

```diff
@@ -0,0 +1,71 @@
+@Component
+public class TaskManager {
+    @PersistenceContext
+    private EntityManager entityManager;
+    private static TaskManager instance;
```

Comment:
why we need this instance? if we have @Component, the scope is single tone by default